content/naorlivne/drone-metronome/index.md

```diff
@@ -0,0 +1,76 @@
+---
+version: '1.0'
+date: 2019-08-27T00:00:00+00:00
+title: Metronome
+author: naorlivne
+tags: [ deploy, metronome ]
+logo: metronome.svg
+repo: naorlivne/drone-metronome
+image: naorlivne/drone-metronome
+---
+
+This plugin can be used to deploy applications to a Metronome server, it will create\update the given Metronome tasks as needed.
+
+The below pipeline configuration demonstrates simple usage:
+
+```yaml
+pipeline:
+name: default
+
+steps:
+- name: metronome_deploy
+  image: naorlivne/drone-metronome
+  settings:
+    metronome_host: http://metronome.mesos:9000
+    metronome_job_file: metronome.json
+```
+
+# Metronome Configuration File
+
+{{% alert info %}}
+In addition to the `.drone.yml` file you will need to create a `metronome.json` file that contains the Metronome configuration. Please see [here](test/test_files/metronome.json) for an example. 
```

Comment:
You should add an absolute path for the example metronome.json config, otherwise this will be a dead link.

Reply:
Pointed to the correct link.